### ISSUE TITLE
5160 ROMs not detected

### ISSUE THREAD

**Issue by dbalsom** [dbalsom/martypc#33]
Yes, they're the same, but the emulator doesn't accept them.

_Originally posted by @chungy in https://github.com/dbalsom/martypc/issues/32#issuecomment-1619155219_

**Comment by dbalsom** [comment]
Issue reported that 5160 ROMs from MAME collection are not recognized during ROM scan.

can you confirm that the ROM md5sums are :
f051b4bbc3b60c3a14df94a0e4ee720f
9696472098999c02217bf922786c1f4a

**Comment by chungy** [comment]
$ md5sum 62x0890.u18 68x4370.u19
    9696472098999c02217bf922786c1f4a  62x0890.u18
    df9f29de490d7f269a6405df1fed69b7  68x4370.u19

**Comment by chungy** [comment]
Oops, I posted the results from a set that works.

    $ md5sum 68x4370.u19
    df9f29de490d7f269a6405df1fed69b7  68x4370.u19

**Comment by dbalsom** [comment]
df9f29de490d7f269a6405df1fed69b7 isn't in that set

**Comment by chungy** [comment]
The `roms.txt` file in `install/roms` lists a combination of ROMs that don't seem to be reflected in `core/src/rom_manager.rs`

Either the code or instructions need to be updated to match.

**Comment by graphixillusion** [comment]
Infact i am using these two roms from the wiki page
```
BIOS_5160_09MAY86_U19_62X0819_68X4370_27256_F000.BIN  **** preferred ****
    32,768 bytes
    md5sum: f051b4bbc3b60c3a14df94a0e4ee720f
-- and --

    BIOS_5160_09MAY86_U18_59X7268_62X0890_27256_F800.BIN  **** preferred ****
    32,768 bytes
    md5sum: 9696472098999c02217bf922786c1f4a
```
but i got this error upon start:

[2023-07-03T18:34:07Z ERROR marty_core::machine] CPU Halted!
[2023-07-03T18:34:07Z ERROR marty_core::machine] CPU Error: The CPU was halted at address: 0FE0AB.

**Comment by dbalsom** [comment]
> Infact i am using these two roms from the wiki page
> 
based on this the emulator detected your roms, can you make sure you are not using Turbo mode, if not please open another issue?

**Comment by dbalsom** [comment]
> The `roms.txt` file in `install/roms` lists a combination of ROMs that don't seem to be reflected in `core/src/rom_manager.rs`
> 
> Either the code or instructions need to be updated to match.

... yep you're right. I must have gotten my hashes crossed writing the documentation.  Sorry. The code should be correct, the ROMs.txt file came much later.

so you need either

                        "fd9ff9cbe0a8f154746ccb0a33f6d3e7", // 5160 BIOS u18 v01/10/86
                        "f051b4bbc3b60c3a14df94a0e4ee720f", // 5160 BIOS u19 v01/10/86

matching MAME ROMS 62x0851.u18 and 62x0854.u19

or                        
                        "9696472098999c02217bf922786c1f4a", // 5160 BIOS u18 v05/09/86
                        "df9f29de490d7f269a6405df1fed69b7", // 5160 BIOS u19 v05/09/86     

matching MAME ROMS 62x0890.u18 and 68x4370.u19

**Comment by graphixillusion** [comment]
> > Infact i am using these two roms from the wiki page
> 
> based on this the emulator detected your roms, can you make sure you are not using Turbo mode, if not please open another issue?

Yep turbo mode is off, and warpspeed too:

turbo = false
warpspeed = false

in toml

**Comment by dbalsom** [comment]
I've updated the Wiki to describe the ROM sets required, and added some more ROM variants to the ROM definition table.

https://github.com/dbalsom/martypc/wiki/ROMs

I've removed the 'roms.txt' file from the repo and just point to the wiki article in the ROM directory readme, so the Wiki should be the definitive info source.

I've confirmed that 0.1.3 will boot using the MAME set.  MartyPC will now print the final set of ROMS it has chosen to use on start-up:

```
Using ROM: ./roms\62x0890.u18
Using ROM: ./roms\68x4370.u19
```
